ui/src/app/tags/components/TagsHeader/AddTagForm/AddTagForm.tsx

```diff
@@ -66,9 +66,9 @@ export const AddTagForm = ({ onClose }: Props): JSX.Element => {
       }}
       onCancel={onClose}
       onSaveAnalytics={{
-        action: "Submit",
-        category: "Create tag form",
-        label: "Create tag",
+        action: "Valid XPath",
+        category: "XPath tagging",
+        label: "Save",
```

Comment:
Wouldn't this be kind of irrelevant for manual tags? Do we need two separate events - one for manual tags and one for automatic?

Reply:
Yes, you're very right, I'll fix that.